### PR TITLE
Remove note on no recovery compression support

### PR DESCRIPTION
**Pull request by jackcarrdev**

### CODE REVIEW

File: docset/winserver2019-ps/dism/Export-WindowsImage.md [hunk 1]
```diff
@@ -71,14 +71,12 @@ Accept wildcard characters: False
...
 - **"max" or "maximum":** Provides the high compression, but takes more time to capture the image
-- **"fast:"** Provides faster image compression, but the resulting files are larger than those compressed by using the maximum option.
+- **"fast":** Provides faster image compression, but the resulting files are larger than those compressed by using the maximum option.
 - **"none":** No compression is used at all. This is the default. 
 
-> [!NOTE]
-> This cmdlet does not support the "recovery" compression type. Use `dism.exe` instead.
-
-the *CompressionType* parameter does not apply when you export an image to an existing .wim file, you can only use this CompressionType when you export an image to a new .wim file.
+The *CompressionType* parameter does not apply when you export an image to an existing .wim file, you can only use this CompressionType when you export an image to a new .wim file.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/dism/Export-WindowsImage.md:79]:
```suggestion
The *CompressionType* parameter does not apply when you export an image to an existing .wim file; you can only use this CompressionType when you export an image to a new .wim file.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Suphax, kindly commit the minor additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]
approving after suggestions applied

**Review by ghost — APPROVED** [review]
Approved pending suggested changes.

**Comment by Suphax** [comment]
Is there anything extra needed for this to be merged? Unsure if I've missed something.

**Comment by Suphax** [comment]
Hello?

**Comment by Suphax** [comment]
Great help thanks

**Pull request closed without merging**